src/pybind/mgr/dashboard/frontend/src/app/core/navigation/identity/identity.component.html

```diff
@@ -0,0 +1,30 @@
+<div dropdown>
+  <a dropdownToggle
+     class="dropdown-toggle"
+     data-toggle="dropdown"
+     i18n-title
+     title="Identity">
```

Comment:
I would prefer the user name as title of the user button :)

Reply:
I thought about this title before I submitted it. Since this drop-down list contains a logout button, I don't know what title will be better, so I use the component name. I will modify it in the new commit.